Add security threat submission beta api docs (#16242)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/security-emailthreatsubmissionpolicy.md

> ---

Reviewer: General for the PR: Please include TOC (Table of Contents) entries for all new entities and its methods.

Author: @isvargasmsft What is the format of the TOC? Is there a sample? Or you can use `suggestion` to provide one sample.
+ @dhairyya

Author: TOC has been added.

title: "emailThreatSubmissionPolicy resource type"

> description: "email threat submission policy"

Reviewer: Description in line 3 (and 16):
1. Usually, these description match. 
2. This description is used by search engines to display results, so put yourself in the shoes of someone performing a search and needs this topic as result. 
3. Preferably, start with a verb in present tense: Represents, Indicates.

A suggestion in this case:
Represents the guidelines to report potential threads and spam in Microsoft Outlook for an organization.

Reviewer: Please improve the description in all topics in this PR.

author: "caigen"
ms.localizationpriority: medium
ms.prod: "security"
doc_type: resourcePageType
---

# emailThreatSubmissionPolicy resource type

Namespace: microsoft.graph.security

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Represents email threat submission policy. It is used for customizing your organization's experience and policy when reporting potential threats and spam in Microsoft Outlook.

Reviewer: Description in lines 3 and 16: In general, we avoid using the name of the resource to explain what the resource does. The second sentence in line 16 is actually good.

In addition, this description is used by search engines to display results. so we try to compose in a way someone would search for this resource in a search engine. 

The current description generates questions like:
In which way can I customize the organization's experience using this resource? It kind of implies that the resource has 2 functions: customize the org experience and set the policy for reporting emails and spam. 

Suggestion:
Represents the guidelines of an organization to report potential threats and spam.

Reviewer: The beta topic still displays the former information. 
<img width="567" alt="image" src="https://user-images.githubusercontent.com/70350956/171038598-ca1d4439-362a-4aed-8762-06a62f89be40.png">

Reviewer: In a previous comment I also suggested: 
Represents the guidelines to report potential threads and spam in Microsoft Outlook for an organization.

> Represents the guidelines of an organization to report potential threats and spam. It is used for customizing your organization's end user threat submission experience when reporting potential threats and spam in Microsoft Outlook Client and OWA.

Reviewer 2: ```suggestion
Represents organizational guidelines about reporting potential threats and spam messages. It is used for customizing your organization's end user threat submission experience when reporting potential threats and spam in Microsoft Outlook.
```

|[Create emailThreatSubmissionPolicy](../api/security-emailthreatsubmissionpolicy-post-emailthreatsubmissionpolicies.md)|[microsoft.graph.security.emailThreatSubmissionPolicy](../resources/security-emailthreatsubmissionpolicy.md)|Create a new [emailThreatSubmissionPolicy](../resources/security-emailthreatsubmissionpolicy.md) object.|
|[Get emailThreatSubmissionPolicy](../api/security-emailthreatsubmissionpolicy-get.md)|[microsoft.graph.security.emailThreatSubmissionPolicy](../resources/security-emailthreatsubmissionpolicy.md)|Read the properties and relationships of an [emailThreatSubmissionPolicy](../resources/security-emailthreatsubmissionpolicy.md) object.|
|[Update emailThreatSubmissionPolicy](../api/security-emailthreatsubmissionpolicy-update.md)|[microsoft.graph.security.emailThreatSubmissionPolicy](../resources/security-emailthreatsubmissionpolicy.md)|Update the properties of an [emailThreatSubmissionPolicy](../resources/security-emailthreatsubmissionpolicy.md) object.|
|[Delete emailThreatSubmissionPolicy](../api/security-emailthreatsubmissionpolicy-delete.md)|None|Deletes an [emailThreatSubmissionPolicy](../resources/security-emailthreatsubmissionpolicy.md) object.|

> ## Properties

Reviewer: Please do not add additional columns to the Properties table. These reference topics follow a template for consistency. In this case, please remove the Key and Required columns.

Reviewer: In addition, please list properties in alphabetical order.

Reviewer: Apply to all docs in this PR.

| Property                                 | Type    | Description                                                                                |
|:-----------------------------------------|:--------|:-------------------------------------------------------------------------------------------|

> | customizedNotificationSenderEmailAddress | String  | Customized notification sender address for review notification message.                    |

Reviewer 2: ```suggestion
| customizedNotificationSenderEmailAddress | String  | The email address of the sender of customized notifications |
```

> | customizedReportRecipientEmailAddress    | String  | When customized address enabled, the message will be sent to this address.                 |

Reviewer 2: ```suggestion
| customizedReportRecipientEmailAddress    | String  | The reported messages from end users will be sent to this address.
```

> | id                                       | String  | Policy id.                                                                                 |

Reviewer 2: ```suggestion
| id                                       | String  | As of today, only one id is supported. The default value is 'DefaultReportSubmissionPolicy'. |
```

> | isAlwaysReportEnabledForUsers            | Boolean | User could use 'Always report the message' option. The default value is true.              |

Reviewer 2: ```suggestion
| isAlwaysReportEnabledForUsers            | Boolean | It is used to specify whether end users can report a message directly without a confirmation. Its default value is true |
```

> | isAskMeEnabledForUsers                   | Boolean | User could use 'Ask me before reporting the message' option. The default value is true.    |

Reviewer 2: ```suggestion
| isAskMeEnabledForUsers                   | Boolean | It is used to specify whether end users can confirm using a popup before reporting messages as spam, phish or not junk. Its default value is true |
```

> | id                                       | String  | Policy id. Only 1 policy is supported by now. The default value is `DefaultReportSubmissionPolicy`. |

Author: ```suggestion
| id                                       | String  | As of today, only one id is supported. The default value is `DefaultReportSubmissionPolicy`. |
```

> | isCustomizedMessageEnabled               | Boolean | If enabled, customized message is enabled.                                                 |

Reviewer 2: ```suggestion
| isCustomizedMessageEnabled               | Boolean | It is used to specify whether the email notifications sent to end users to inform them whether an email is phish, spam or junk is customized or not? |
```

> | isCustomizedMessageEnabledForPhishing    | Boolean | If enabled, customized message only shows when email is reported as phishing.              |

Reviewer 2: ```suggestion
| isCustomizedMessageEnabledForPhishing    | Boolean | It is used to specify whether the email notifications sent to end users is to be sent for phish or not? |
```

> | isCustomizedNotificationSenderEnabled    | Boolean | If enabled, notificationSenderAddress is used by review notification email.                |

Reviewer 2: ```suggestion
| isCustomizedNotificationSenderEnabled    | Boolean | It is used to specify whether we should use the sender email address which we set earlier for sending email notifications to end users or not?|
```

> | isNeverReportEnabledForUsers             | Boolean | User could use 'Never report the message' option. The default value is true.               |

Reviewer 2: ```suggestion
| isNeverReportEnabledForUsers             | Boolean |  It is used to specify whether end users can simply move the message from one folder to another based on the action of spam, phish or not junk without actually reporting it. Its default value is true |
```

> | isReportFromQuarantineEnabled            | Boolean | If enabled, submission could be created from security portal quarantine page.              |

Reviewer 2: ```suggestion
| isReportFromQuarantineEnabled            | Boolean | It is used to specify whether end users can submit from the quarantine page or not?  |
```

> | isReportToCustomizedEmailAddressEnabled  | Boolean | If enabled, the email will be sent to customizedReportRecipientEmailAddress.               |

Reviewer 2: ```suggestion
| isReportToCustomizedEmailAddressEnabled  | Boolean | It is used to specify whether emals reported by end users should be send to the custom mailbox which was configured above using customizedReportRecipientEmailAddress  |
```

> | isReportToMicrosoftEnabled               | Boolean | If enabled, the email will be sent to Microsoft for analysis.                              |

Reviewer 2: ```suggestion
| isReportToMicrosoftEnabled               | Boolean |It is used to specify whether the email messages reported by end users as junk, phish or not junk be sent to Microsoft or not? |
```

> | isReviewEmailNotificationEnabled         | Boolean | If enabled, system sends email notification to reporter when admin reviews the submission. |

Reviewer 2: ```suggestion
| isReviewEmailNotificationEnabled         | Boolean | It is used to specify whether an email notification be sent to the end user who reported the email when it has been reviewed by the Admin? |
```

> | isOrganizationBrandingEnabled            | Boolean | If enabled, tenant branding logo will be used in review notification message.              |

Reviewer: Please order these properties alphabetically.

Reviewer: Regarding the description of the properties, please take in consideration 2 main principles:
1. We avoid constructions with 3 or more nouns in a row because they are hard to understand for readers whose native language is not English. For example: customized notification sender email address, review notification email/message, security portal quarantine page. In such cases, a better composition would be: The email address of the sender of customized notifications, an email notification of a review/for a review, the quarantine page in the security portal.
2. It is important to put yourself in the shoes of the developer who is reading and trying to use the API. When we say " the email address for review notification message" it's not clear what is the email for. Does it receive notification messages? Does it send notification messages? What's the functionality? 
Similar to "When customized address enabled", what is customized address and how to you enable it? Are we referring to a property of this resource? In that case, its better to call the property by its actual name. 
Another example when we say "If enabled, notificationSenderAddress is used to ..." what is notificationSenderAddress? Is it a property? a resource type? If it is a resource type then we should be including a reference link to its topic. 

Please note the changes made to this table should be reflected in the tables in the Create and Update topics as well.

Feel free to ping me on the side if you have questions around these property descriptions.

Author: @dhairyya This one.

Author: Resolving this one since it is outdated. Some sentences have been refined.

> | isOrganizationBrandingEnabled            | Boolean | If enabled, tenant branding logo will be used in review notification message.              |

Reviewer 2: ```suggestion
| isOrganizationBrandingEnabled            | Boolean | It is used to specify whether the branding logo should be used in the email notifications or not which are being sent to end users? |
```

> | isOrganizationBrandingEnabled            | Boolean | If enabled, tenant branding logo will be used in review notification message. The default value is `false`. Optional for creation.        |

Author: ```suggestion
| isOrganizationBrandingEnabled            | Boolean | It is used to specify whether the branding logo should be used in the email notifications or not which are being sent to end users? The default value is `false`. Optional for creation.        |
```

> | customizedNotificationSenderEmailAddress | String  | Customized notification sender address for review notification message. The default value is `null`. Optional for creation.                   |
> | customizedReportRecipientEmailAddress    | String  | When customized address enabled, the message will be sent to this address. The default value is `null`. Optional for creation. |
> | id                                       | String  | Policy id. Only 1 policy is supported by now. The default value is `DefaultReportSubmissionPolicy`. |
> | isAlwaysReportEnabledForUsers            | Boolean | User could use 'Always report the message' option. The default value is `true`.  Optional for creation.          |
> | isAskMeEnabledForUsers                   | Boolean | User could use 'Ask me before reporting the message' option. The default value is true. Optional for creation.   |
> | isCustomizedMessageEnabled               | Boolean | If enabled, customized message is enabled. The default value is `false`. Optional for creation.                  |
> | isCustomizedMessageEnabledForPhishing    | Boolean | If enabled, customized message only shows when email is reported as phishing. The default value is `false`. Optional for creation. |
> | isCustomizedNotificationSenderEnabled    | Boolean | If enabled, notificationSenderAddress is used by review notification email. The default value is `false`. Optional for creation.               |
> | isNeverReportEnabledForUsers             | Boolean | User could use 'Never report the message' option. The default value is `true`. Optional for creation.         |
> | isOrganizationBrandingEnabled            | Boolean | If enabled, tenant branding logo will be used in review notification message. The default value is `false`. Optional for creation.        |
> | isReportFromQuarantineEnabled            | Boolean | If enabled, submission could be created from security portal quarantine page. The default value is `true`. Optional for creation.              |
> | isReportToCustomizedEmailAddressEnabled  | Boolean | If enabled, the email will be sent to customizedReportRecipientEmailAddress.  The default value is `false`. Optional for creation.              |
> | isReportToMicrosoftEnabled               | Boolean | If enabled, the email will be sent to Microsoft for analysis. The default value is `false`. Requried for creation.  |
> | isReviewEmailNotificationEnabled         | Boolean | If enabled, system sends email notification to reporter when admin reviews the submission. The default value is `false`. Optional for creation.  |

Author: ```suggestion
| customizedNotificationSenderEmailAddress | String  | It is used to specify the email address of the sender from which email notifications will be sent to end users to inform them whether an email is spam, phish or clean. The default value is `null`. Optional for creation.                   |
| customizedReportRecipientEmailAddress    | String  | It is used to specify the destination where the reported messages from end users will land whenever they report something as phish, junk or not junk. The default value is `null`. Optional for creation. |
| id                                       | String  | Policy id. Only 1 policy is supported by now. The default value is `DefaultReportSubmissionPolicy`. |
| isAlwaysReportEnabledForUsers            | Boolean | It is used to specify whether end users can report a message as spam, phish or junk directly without a confirmation(popup). The default value is `true`.  Optional for creation.          |
| isAskMeEnabledForUsers                   | Boolean | It is used to specify whether end users can confirm using a popup before reporting messages as spam, phish or not junk. The default value is `true`.  Optional for creation.   |
| isCustomizedMessageEnabled               | Boolean | It is used to specify whether the email notifications sent to end users to inform them whether an email is phish, spam or junk is customized or not? The default value is `false`. Optional for creation.                  |
| isCustomizedMessageEnabledForPhishing    | Boolean | It is used to specify whether the customized email notifications sent to end users is to be sent for phish or not. The default value is `false`. Optional for creation. |
| isCustomizedNotificationSenderEnabled    | Boolean | It is used to specify whether we should use the sender email address which we set earlier using customizedNotificationSenderEmailAddress for sending email notifications to end users or not? The default value is `false`. Optional for creation.               |
| isNeverReportEnabledForUsers             | Boolean | It is used to specify whether end users can simply move the message from one folder to another based on the action of spam, phish or not junk without actually reporting it. The default value is `true`. Optional for creation.         |
| isOrganizationBrandingEnabled            | Boolean | If enabled, tenant branding logo will be used in review notification message. The default value is `false`. Optional for creation.        |
| isReportFromQuarantineEnabled            | Boolean | It is used to specify whether end users can submit from the quarantine page or not? The default value is `true`. Optional for creation.              |
| isReportToCustomizedEmailAddressEnabled  | Boolean | It is used to specify whether emails reported by end users should be send to the custom mailbox which was configured above using customizedReportRecipientEmailAddress.  The default value is `false`. Optional for creation.              |
| isReportToMicrosoftEnabled               | Boolean | It is used to specify whether the email messages reported by end users as junk, phish or not junk be sent to Microsoft or not? The default value is `false`. Requried for creation.  |
| isReviewEmailNotificationEnabled         | Boolean | It is used to specify whether an email notification be sent to the end user who reported the email when it has been reviewed by the Admin? The default value is `false`. Optional for creation.  |
```

> | isCustomizedMessageEnabled               | Boolean | It is used to specify whether the email notifications sent to end users to inform them whether an email is phish, spam or junk is customized or not? The default value is `false`. Optional for creation.                  |
> | isCustomizedMessageEnabledForPhishing    | Boolean | It is used to specify whether the customized email notifications sent to end users is to be sent for phish or not. The default value is `false`. Optional for creation. |
> | isCustomizedNotificationSenderEnabled    | Boolean | It is used to specify whether we should use the sender email address which we set earlier using customizedNotificationSenderEmailAddress for sending email notifications to end users or not? The default value is `false`. Optional for creation.               |
> | isNeverReportEnabledForUsers             | Boolean | It is used to specify whether end users can simply move the message from one folder to another based on the action of spam, phish or not junk without actually reporting it. The default value is `true`. Optional for creation.         |
> | isOrganizationBrandingEnabled            | Boolean | It is used to specify whether the branding logo should be used in the email notifications or not which are being sent to end users? The default value is `false`. Optional for creation.        |
> | isReportFromQuarantineEnabled            | Boolean | It is used to specify whether end users can submit from the quarantine page or not? The default value is `true`. Optional for creation.              |
> | isReportToCustomizedEmailAddressEnabled  | Boolean | It is used to specify whether emails reported by end users should be send to the custom mailbox which was configured above using customizedReportRecipientEmailAddress.  The default value is `false`. Optional for creation.              |
> | isReportToMicrosoftEnabled               | Boolean | It is used to specify whether the email messages reported by end users as junk, phish or not junk be sent to Microsoft or not? The default value is `false`. Requried for creation.  |
> | isReviewEmailNotificationEnabled         | Boolean | It is used to specify whether an email notification be sent to the end user who reported the email when it has been reviewed by the Admin? The default value is `false`. Optional for creation.  |

Author: ```suggestion
| isCustomizedMessageEnabled               | Boolean | It is used to specify whether the email notifications sent to end users to inform them whether an email is phish, spam or junk is customized or not. The default value is `false`. Optional for creation.                  |
| isCustomizedMessageEnabledForPhishing    | Boolean | It is used to specify whether the customized email notifications sent to end users is to be sent for phish or not. The default value is `false`. Optional for creation. |
| isCustomizedNotificationSenderEnabled    | Boolean | It is used to specify whether we should use the sender email address which we set earlier using customizedNotificationSenderEmailAddress for sending email notifications to end users or not. The default value is `false`. Optional for creation.               |
| isNeverReportEnabledForUsers             | Boolean | It is used to specify whether end users can simply move the message from one folder to another based on the action of spam, phish or not junk without actually reporting it. The default value is `true`. Optional for creation.         |
| isOrganizationBrandingEnabled            | Boolean | It is used to specify whether the branding logo should be used in the email notifications or not which are being sent to end users. The default value is `false`. Optional for creation.        |
| isReportFromQuarantineEnabled            | Boolean | It is used to specify whether end users can submit from the quarantine page or not. The default value is `true`. Optional for creation.              |
| isReportToCustomizedEmailAddressEnabled  | Boolean | It is used to specify whether emails reported by end users should be send to the custom mailbox which was configured above using customizedReportRecipientEmailAddress.  The default value is `false`. Optional for creation.              |
| isReportToMicrosoftEnabled               | Boolean | It is used to specify whether the email messages reported by end users as junk, phish or not junk be sent to Microsoft or not. The default value is `false`. Requried for creation.  |
| isReviewEmailNotificationEnabled         | Boolean | It is used to specify whether an email notification be sent to the end user who reported the email when it has been reviewed by the Admin. The default value is `false`. Optional for creation.  |
```
